person/tests_asem_user.py

```diff
@@ -894,4 +1101,279 @@ def test_asem_user_create_bank_account_number_incorrect_pattern(self):
             family_unit_size=1,
             own_home="VP",
             own_vehicle=True,
-            bank_account_number='No es un patron de cuenta bancaria',ong=self.ong)
+            bank_account_number='ES11111111111111111111111',
+            ong=None)
+
+    def test_asem_user_create_ong_incorrect_type(self):
+        with self.assertRaises(Exception):
+            ASEMUser.objects.create(email="tcamerob3@gmail.com",
+            name="Tomas2",
+            surname='Camero',
+            birth_date=datetime.datetime(2000, 1, 24, tzinfo=datetime.timezone.utc),
+            sex="M",
+            city="Sevilla",
+            address="Logroño 19",
+            telephone=691644398,
+            postal_code=41730,
+            condition="EM",
+            member="UNA",
+            user_type="OTROS",
+            correspondence="CC",
+            status="F",
+            family_unit_size=1,
+            own_home="VP",
+            own_vehicle=True,
+            bank_account_number='ES11111111111111111111111',
+            ong="No es una ONG")
+
+    ### TESTS UPDATE ASEM USER ###
+
+    def test_asem_user_update_name_incorrect_max_length(self):
+        with self.assertRaises(Exception):
+            self.asem_user_update.name = "M" * 51
+            self.asem_user_update.save()
+
+    def test_asem_user_update_name_incorrect_null(self):
+        with self.assertRaises(Exception):
+            self.asem_user_update.name = None
+            self.asem_user_update.save()
+
+    def test_asem_user_update_surname_incorrect_max_length(self):
+        with self.assertRaises(Exception):
+            self.asem_user_update.surname = "M" * 51
+            self.asem_user_update.save()
+
+    def test_asem_user_update_surname_incorrect_null(self):
+        with self.assertRaises(Exception):
+            self.asem_user_update.surname = None
+            self.asem_user_update.save()
+
+    def test_asem_user_update_email_incorrect(self):
+        with self.assertRaises(Exception):
+            self.asem_user_update.email = "emailIncorrecto"
+            self.asem_user_update.full_clean()
+
+    def test_asem_user_update_email_incorrect_null(self):
+        with self.assertRaises(Exception):
+            self.asem_user_update.email = None
+            self.asem_user_update.save()
+
+    def test_asem_user_update_birth_date_incorrect_null(self):
+        with self.assertRaises(Exception):
+            self.asem_user_update.birth_date = None
+            self.asem_user_update.sfull_clean()
```

Comment:
```suggestion
            self.asem_user_update.full_clean()
```

Reply:
I hadn't noticed that error, I fixed it and asked you for a review again. Thank you very much Jaime! 🙌🏻 💪🏻